airflow/providers/apache/kafka/hooks/consumer.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import Any, Sequence
+
+from confluent_kafka import Consumer
+
+from airflow.exceptions import AirflowException
+from airflow.hooks.base import BaseHook
+
+
+class KafkaConsumerHook(BaseHook):
+    """KafkaConsumerHook
+
+    A hook for creating a Kafka Consumer
+    :param config: A config dictionary to use with confluent_kafka library, defaults to None
```

Comment:
```suggestion
    A hook for creating a Kafka Consumer

    :param config: A config dictionary to use with confluent_kafka library, defaults to None
```
The blank line is needed between the description and parameter directives so the parameters are properly rendered in the Python API docs. This comment applies to other instances in the provider as well.